drivers/include/net/netdev.h

```diff
@@ -275,9 +275,7 @@ struct netdev {
     const struct netdev_driver *driver;     /**< ptr to that driver's interface. */
     netdev_event_cb_t event_callback;       /**< callback for device events */
     void* context;                          /**< ptr to network stack context */
-#ifdef MODULE_NETSTATS_L2
-    netstats_t stats;                       /**< transceiver's statistics */
-#endif
+    netdev_t *child;                        /**< ptr to the lower netdev layer */
```

Comment:
Somehow I'd choose parent as name, don't now why. Any reasoning for "child"?

Reply:
I think this is due to my mental image of the layered stack as a literal stack of layers, the bottom one being the hardware driver and everything on top adding functionality. A packet being transmitted is handed down starting at the upper most layer to the layer below it, thus, from the parent layer to the child layer.

If you think parent (or upper/lower as stated below) is more clear to other people, I don't mind changing it. The goal here is of course to have something that is as intuitive as possible.